u2f-tests/NFC/u2f_nfc_util.c

```diff
@@ -307,15 +316,18 @@ uint xchgAPDUExtended(uint cla, uint ins, uint p1, uint p2, uint lc, const void
   capdu[8+lc]=(uint8_t) (*rapduLen & 0xff);
   len = lc+9;
 
-
-
   printCmdAPDU(capdu, len);
   rc = SCardTransmit(hCard, SCARD_PCI_T1, capdu, len, NULL, (uint8_t*) rapdu, &rlen );
   if (!check("SCardTransmit (3)", rc)) return PCSC_ERROR;
   printRespAPDU((uint8_t*) rapdu, rlen);
-
+ 
+  if(rlen >= 2){
```

Comment:
Alexei – If I look at the style guide I see:

if (condition) {  // no spaces inside parentheses


Followed by an example that includes spaces inside the parenthesis:

if (x == kFoo) return new Foo();


I’m assuming no space at all inside the parentheses?


-          C


From: Alexei [mailto:notifications@github.com]
Sent: Wednesday, December 02, 2015 11:26 AM
To: google/u2f-ref-code
Cc: Woodthorpe Christopher (IFAM CCS SMD AMR)
Subject: Re: [u2f-ref-code] Added Kh size tests (#62)


In u2f-tests/NFC/u2f_nfc_util.c<https://github.com/google/u2f-ref-code/pull/62#discussion_r46461765>:

>    printCmdAPDU(capdu, len);

>    rc = SCardTransmit(hCard, SCARD_PCI_T1, capdu, len, NULL, (uint8_t*) rapdu, &rlen );

>    if (!check("SCardTransmit (3)", rc)) return PCSC_ERROR;

>    printRespAPDU((uint8_t*) rapdu, rlen);

> -

> +

> +  if(rlen >= 2){

please format as per style guide:
https://google.github.io/styleguide/cppguide.html#Conditionals

—
Reply to this email directly or view it on GitHub<https://github.com/google/u2f-ref-code/pull/62/files#r46461765>.